src/__test__/plugin.test.js

```diff
@@ -545,6 +545,39 @@ describe("draft-js-markdown-plugin", () => {
                 " "
               );
             });
+            it("unstickys inline style", () => {
+              currentRawContentState = {
+                entityMap: {},
+                blocks: [
+                  {
+                    key: "item1",
+                    text: "item1",
+                    type: "unstyled",
+                    depth: 0,
+                    inlineStyleRanges: [
+                      { offset: 0, length: 5, style: "BOLD" },
+                    ],
+                    entityRanges: [],
+                    data: {},
+                  },
+                ],
+              };
+
+              currentSelectionState = currentSelectionState.merge({
+                focusOffset: 5,
+                anchorOffset: 5,
+              });
+
+              expect(subject()).toBe("handled");
+              expect(store.setEditorState).toHaveBeenCalled();
+              newEditorState = store.setEditorState.mock.calls[0][0];
+              const block = newEditorState.getCurrentContent().getLastBlock();
+              const length = block.getLength();
+              expect(block.getInlineStyleAt(length - 1).toJS()).toEqual([]);
+              expect(block.getInlineStyleAt(length - 2).toJS()).toEqual([
+                "BOLD",
+              ]);
+            });
```

Comment:
I'd also add some unit tests for when stickiness doesn't apply: like when you're in an entity for example :+1: just to make sure it doesn't break in future